### DISCUSSION TITLE
I'm not quite clear on how this works

### DISCUSSION THREAD

**Discussion by HyperCrab2000** [justkey007/tsc-alias#180] [Q&A]
It looks like tsc-alias goes through all of the built files and replaces absolute paths with relative paths, right?

But what if tsc before running tsc-alias gives you lots of 'cannot find module' errors? Should we ignore these? Will tsc still build everything out if it cannot resolve the modules? 

If these errors from tsc should be ignored, then how can we bypass this for pipeline steps? Or is there a way for tsc to not spit these errors out so that it can immediately go to tsc-alias?

**Comment by HyperCrab2000** [comment]
So I made a mistake in my tsconfig file and updated it accordingly and now I no longer see the tsc errors. But without any errors, do I still need tsc-alias to update the paths?

**Reply by justkey007** [reply]
tsc-alias has the function of replacing path aliases with relative paths after typescript compilation. If you are using path aliases with typescript I believe you must need them.

**Reply by HyperCrab2000** [reply]
Even if tsc has no errors on output? I'll double check in a bit but there's one easy way to test it at least.
